src/shared-components/carousel/arrow/index.js

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import { ArrowLeftIcon, ArrowRightIcon } from 'src/icons';
+import { COLORS } from 'src/constants';
+
+import ArrowContainer from './style';
+
+const Arrow = props => {
+  const {
+    style,
+    onClick,
+    onUserInteraction,
+    prev,
+    next,
+    disabled,
+    ...rest
+  } = props;
+
+  return (
+    <ArrowContainer
+      {...rest}
+      style={{ ...style }}
```

Comment:
were we using this `style` prop before? We typically use the `css` prop for emotion and never the `style` prop.

Reply:
I dont think we even need that, I just removed the prop and still working fine